### ISSUE TITLE
Dropdown: multiple `defaultValue` & `value` have incorrect types

### ISSUE THREAD

**Issue by zhongeric** [Semantic-Org/Semantic-UI-React#4017]
## Bug Report

Just stumbled across this - not sure if it's a bug or an user error.

### Steps
A clear and concise description of steps to reproduce the problem.

Reproduce:

`<Dropdown
        placeholder="Add tags"
                                fluid
                                multiple
                                selection
                                options={this.state.allTagsFormatted}
                                onChange={this.handleTagChange}
                                defaultValue={['VALUE1', 'VALUE2']}
/>`

Where this.state.allTagsFormatted is an array of type `key: string;
    text: string;
    value: string;
    created_date: string;`

And passing an array of strings of options values into the defaultValue prop. Example: ['1111','2222', ...]

### Expected Result
The result that you expected.

Compiles and pass type check per the [docs](https://react.semantic-ui.com/modules/dropdown/#types-multiple-selection): 
`

defaultValue |   | {number\|string\|bool\|arrayOf} | Initial value or value array if multiple.
-- | -- | -- | --

`

### Actual Result
The actual result that happened 💣

Throws this typescript error: 
`Type 'string' is not assignable to type 'boolean'`

### Version

semantic-ui-react : "^1.1.0",
"typescript": "^3.9.7"
"@craco/craco": "^5.6.4",
"@semantic-ui-react/craco-less": "^1.2.1",
"semantic-ui-less": "^2.4.1"

### Testcase
[Fork, update, and replace this pen to show the bug]:
https://codesandbox.io/s/2l3n74j9y

**Comment by layershifter** [comment]
Thanks for reporting 👍 It's the same as #4013 i.e. typings issue 😿 

---

`src/modules/Dropdown/Dropdown.d.ts`

```diff
  /** Initial value or value array if multiple. */
-  defaultValue?: string | number | boolean | number | string | boolean[]
+  defaultValue?: string | number | boolean | (number | string | boolean)[]
```

```diff
  /** Current value or value array if multiple. Creates a controlled component. */
-  value?: boolean | number | string | boolean | number | string[]
+  value?: boolean | number | string | (boolean | number | string)[]
```

Would you like to a submit a PR to fix the issue? 😺

**Comment by zhongeric** [comment]
https://github.com/Semantic-Org/Semantic-UI-React/pull/4018 @layershifter 

Just submitted one, can't get past the label checks though haha

**Comment by xumix** [comment]
@layershifter Hi! Any news on this? I'd really like to update to newer version of the lib.

**Comment by layershifter** [comment]
@xumix released in `1.2.0`, please check 🙏

**Comment by xumix** [comment]
> @xumix released in `1.2.0`, please check 🙏

Thanks a lot!
